tests/api_connexion/endpoints/test_dag_run_endpoint.py

```diff
@@ -307,6 +336,41 @@ def test_should_respond_200(self, session):
             "total_entries": 2,
         }
 
+    def test_filter_by_state(self, session):
+        self._create_test_dag_run_with_queued()
+        assert session.query(DagRun).count() == 2
+        response = self.client.get(
+            "api/v1/dags/TEST_DAG_ID/dagRuns?state=running,queued", environ_overrides={'REMOTE_USER': "test"}
+        )
+        assert response.status_code == 200
+        assert response.json == {
```

Comment:
You may not need to check and list every fields. You can check important things like total returned, and states of the items returned

Reply:
Hi @ephraimbuddy, OK understood. I can change that according to recommendation. Thanks.